src/commands/call.js

```diff
@@ -15,8 +15,9 @@ class CallCommand extends ApifyCommand {
         const localConfig = getLocalConfig() || {};
         const runOpts = {
             actId: args.actId || localConfig.actId,
-            waitForFinish: 120,
+            waitForFinish: 2,
```

Comment:
Why 2? Can we omit this option?

Reply:
Sometimes, when I omitted waitForFinish, I can not reach stream because I try to connect to it before it was created